lib/chef/provisioning/docker_driver/driver.rb

```diff
@@ -70,66 +70,116 @@ def self.connection_url(driver_url)
       end
     end
 
-
     def allocate_machine(action_handler, machine_spec, machine_options)
```

Comment:
Hey @mwrock - now that https://github.com/chef/chef-provisioning/pull/366 is merged you should just be able to use `from_image = machine_options.from_image` correct?